doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -267,6 +267,7 @@ integration with 3rd party tools like logstash.
       #redis:
       #  server: 127.0.0.1
       #  port: 6379
+      #  async: true ## if redis replies are read asynchronously
```

Comment:
I thought this was on the wiki but its not, documentation commits should be in their own commit please, separate from code commits.